proarc-common/src/main/java/cz/cas/lib/proarc/common/fedora/AbstractFedoraObject.java

```diff
@@ -56,4 +56,8 @@ public void flush() throws DigitalObjectException {
         }
     }
 
+    @Override
+    public void purgeDatastream(String datastream, String logMessage) throws DigitalObjectException {
+        RemoteStorage.getInstance().find(pid).purgeDatastream(datastream, logMessage);
```

Comment:
Remote purge datastream nemůže být společná implementace pro remote a local object

Reply:
Tady jsem použil tuto implementaci, protože mi z JavaDocu LocalStorage nebylo jasné, co se má v tomto případě použít namísto `FedoraClient.purgeDatastream()`, vzhledem k uložení v paměti.

Reply:
Popis viz. výše. Implementace je zřejmá z LocalObject. I javadoc říká `It contains newly imported but not yet ingested files.` Tedy remote neexistuje pokud není ingested :wink:
https://github.com/proarc/proarc/blob/fd6149e39492baf7a3a40169f36e420a218497cc/proarc-common/src/main/java/cz/cas/lib/proarc/common/fedora/LocalStorage.java#L54-L57

Reply:
Předpokládám, že tedy také stačí UnsupportedOpEx.

Reply:
AbstractFedoraObject.purgeDatastream tam být nemusí vůbec, ale UOE je taky v pohodě.